express-static/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+const join = path.join('public', '__dirname');
+console.log(join);
+
+const staticReturn = app.use(express.static('public'));
```

Comment:
Here you don't need to save the returned value from `app.use` also you want to pass the file path you created above to `static`:
```JAVASCRIPT
app.use(express.static(publicPath));
```